xrpl/wallet/main.py

```diff
@@ -77,8 +112,88 @@ def create(
         Returns:
             The wallet that is generated from the given seed.
         """
-        seed = generate_seed(algorithm=crypto_algorithm)
-        return cls(seed, sequence=0, algorithm=crypto_algorithm)
+        return cls(algorithm=crypto_algorithm)
+
+    @classmethod
+    def from_public_private_keys(
+        cls: Type[Wallet],
+        public_key: str,
+        private_key: str,
+        master_address: Optional[str] = None,
+    ) -> Wallet:
+        """
+        Generates a new Wallet from public and private keys.
+
+        Args:
+            public_key: The public key for the account.
+            private_key: The private key used for signing transactions for the account.
+            master_address: Include if a Wallet uses a Regular Key Pair. It must be
+                the master address of the account. The default is `None`.
+
+        Returns:
+            The wallet that is generated from the given keys.
+        """
+        wallet = cls()
+        wallet.seed = None
+        wallet.public_key = public_key
+        wallet.private_key = private_key
+        wallet.classic_address = (
+            ensure_classic_address(master_address)
+            if master_address is not None
+            else derive_classic_address(public_key)
+        )
+        return wallet
```

Comment:
Instead of doing this (which is kind of hacky), what do you think about essentially an alternate constructor?
```py
def __init__(
        self: Wallet,
        seed: Optional[str] = None,
        sequence: Optional[int] = 0,
        *,
        public_key: Optional[str] = None,
        private_key: Optional[str] = None,
        algorithm: Optional[CryptoAlgorithm] = None,
        master_address: Optional[str] = None,
    ) -> None
    # it's invalid for both `seed` and `public_key`/`private_key` to be passed in - throw an error if that's the case
```

Reply:
All of this logic would go in the same constructor for the wallet? Not 2 separate constructors?

Reply:
Python doesn't really support alternate constructor in the normal/Java sense.

Reply:
I think this method might be a bit confusing for developers